src/processTargets/modifiers/processModifier.ts

```diff
@@ -55,12 +54,8 @@ export default function (
       break;
 
     case "toRawSelection":
-      result = processRawSelectionModifier(context, selection, modifier);
+      result = processRawSelectionModifier(context, selection);
       break;
-
-    default:
-      // Make sure we haven't missed any cases
-      const neverCheck: never = modifier;
```

Comment:
Why have you removed this?  It is enables the type checker to enforce that we haven't missed a case.  Maybe just rename it to `_neverCheck`?

Reply:
I don't understand this syntax at all. To my eyes you're just creating unused variable with no purpose. Could you point me to some documentation?

Reply:
https://www.typescriptlang.org/docs/handbook/2/narrowing.html#exhaustiveness-checking